recipes/fast-double-parser/all/conanfile.py

```diff
@@ -0,0 +1,36 @@
+import os
+from conans import ConanFile, tools
+
+required_conan_version = ">=1.43.0"
+
+
+class FastDoubleParserConan(ConanFile):
+    name = "fast-double-parser"
```

Comment:
```suggestion
    name = "fast_double_parser"
```

Please, follow the upstream's name. You also need to rename the folder name.

Reply:
I fixed it. Thank you a lot!